app/src/main/java/nomic/data/models/GameDTO.kt

```diff
@@ -0,0 +1,20 @@
+package nomic.data.models
+
+import java.time.LocalDate
+
+/**
+ * Data class for game
+ *
+ * @property gameId the unique id of the game
+ * @property title the display title of a game
+ * @property createDate the date when the game was created
+ * @property currentPlayer the indicator of who's turn it is
+ * @property userId the user the game is attached to
+ */
+data class GameDTO(
+    val gameId: Int? = null,
+    val title: String,
+    val createDate: LocalDate? = null,
```

Comment:
Not from the backend, but to the backend it can be since create a game only utilizes data for userId and title
![image](https://user-images.githubusercontent.com/57193675/233516365-c8a1063a-4a24-4b89-9c38-5fb32c6015bb.png)